### DISCUSSION TITLE
Onion V4.2.0.1-beta Download Location Missing?

### DISCUSSION THREAD

**Discussion by jaxform** [OnionUI/Onion#907] [Q&A]
Hello, 

I was looking for the download for the Onion V4.2.0.1-beta download under the releases page and I cannot seem to find the zip file for the OS. Under the assets for the newly hyperlinked page (now "latest") there are only 2 things, both being source codes. I was just looking for some help to locate the download page so that I may install the system on my Miyoo Mini Plus+.

![image](https://github.com/OnionUI/Onion/assets/136674633/b801d963-f0cf-4983-90f5-a679a705c771)
![image](https://github.com/OnionUI/Onion/assets/136674633/46dd6b5d-8916-47c4-9190-a8d0d8c6b995)

**Comment by AahilD** [comment]
Looks like there was an issue with the release

https://github.com/OnionUI/Onion/discussions/747#discussioncomment-6178760
